Index: src/Compass.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Compass.cs b/src/Compass.cs
new file mode 100644
--- /dev/null	(date 1716586663913)
+++ b/src/Compass.cs	(date 1716586663913)
@@ -0,0 +1,18 @@
+using Godot;
+
+public class Compass : Sprite {
+    
+    private Sprite foreground;
+
+    public override void _Ready() {
+        foreground = GetNode("Compass-fg") as Sprite;
+    }
+
+    public void setWindDirection(Vector2 windDir) {
+        foreground.Rotation = windDir.AngleTo(Vector2.Up);
+    }
+
+    public override void _Process(float delta) {
+        setWindDirection(new Vector2((float)GD.RandRange(-1f, 1f), (float) GD.RandRange(-1, 1)));
+    }
+}
\ No newline at end of file
